### PR TITLE
Bug 1463814: crontabber should spit out version.json on startup

### PR DESCRIPTION
**Pull request by is2ei**
This issue is to make crontabber to spit out the content of `version.json`.
This PR changes crontabber to spit out the content of `version.json` if the file exists.

You can check it by:
1. Provide version.json 
2. Run crontabber.

### CODE REVIEW

File: socorro/cron/crontabber_app.py [hunk 1]
```diff
@@ -177,6 +184,11 @@ def mark_success(self, job_list_string):
             self._remember_success(job_class, success_date=now, duration=0)
 
     def main(self):
+        version_json_path = os.path.join(ROOT, "version.json")
+        if os.path.exists(version_json_path):
+            with open(version_json_path, 'r') as fp:
+                self.config.logger.info('version.json: ' + fp.read().strip())
+
```
Comment (willkg) [inline socorro/cron/crontabber_app.py]: I fiddled with the code a bit and I think my idea of putting this in `SocorroApp` is a better solution. It covers both crontabber and processor apps so we don't need to do this twice. It also prints out the version information before all the configuration information. That's pretty handy.

Can you move it there, please?

Comment (is2ei) [reply socorro/cron/crontabber_app.py]: @willkg Thanks for reviewing!
I moved the function. Could you check it, please?

File: socorro/app/socorro_app.py [hunk 2]
```diff
@@ -202,6 +215,7 @@ def fix_exit_code(code):
                     respond_to_SIGHUP,
                     logger=config.logger
                 )
+                spit_out_version_json(config.logger)
```
Comment (willkg) [inline socorro/app/socorro_app.py]: It helps me if the version information is logged before all the config information. I'd move this line above the `try` block. Can you do that, please?

Comment (is2ei) [reply socorro/app/socorro_app.py]: @willkg Sure! I fixed it. Could you check it, please?

**Pull request merged**